video-prop-guidelines (#1098)
demisto/content-docs: docs/documentation/pack-docs.md

Content Packs displayed in the Cortex XSOAR Marketplace contain 2 main documentation sections: 
* **Description**: displayed in the Content Pack card when browsing the Marketplace and in the top of the *Details* tab.

> * **Video**: displayed in the main display area and in the middle of the *Details* tab.

Author: ```suggestion
* **Videos**: displayed in the main display area and in the middle of the *Details* tab.
```

> * **README**: displayed in the main display area and in the bottom of the *Details* tab.

Reviewer: Video or Videos?

* "Eliminate ________ by improving your__________. Rich with layouts and playbooks, this Content Pack is right for ____________"
* "Get smarter. This pack utilizes _________ and _______for when _________ is heavily needed"

> ## Pack Video

Reviewer: Same here...

> ## Pack Video

Author: ```suggestion
## Pack Videos
```

> For larger packs that provide at least one end-to-end use case you are encouraged to create a short video for the pack that will be showed in the *Details* tab of the pack in the Marketplace. The video file should hosted in youtube. The video should contain a more detailed overview of the Pack compared to the *Description* section, but stay tuned to the value proposition of the pack.

Author: ```suggestion
For larger packs that provide at least one end-to-end use case you are encouraged to create a short video or a few videos for the pack that will be showed in the *Details* tab of the pack in the Marketplace. The videos files should be hosted in youtube. The videos should contain a more detailed overview of the Pack compared to the *Description* section, but stay tuned to the value proposition of the pack.
```
